lib/upsert/merge_function.rb

```diff
@@ -31,12 +31,13 @@ def unique_name(table_name, selector_keys, setter_keys)
     attr_reader :selector_keys
     attr_reader :setter_keys
 
-    def initialize(controller, selector_keys, setter_keys, assume_function_exists)
+    def initialize(controller, selector_keys, setter_keys, assume_function_exists, use_native_upsert = nil)
```

Comment:
@seamusabshere I'll check the flow through, but I was using an optional argument because sometimes you don't care whether it's being used or not ;)